04_numeronym/starfallz/main.go

```diff
@@ -0,0 +1,33 @@
+package main
+
+import (
+	"fmt"
+	"io"
+	"os"
+	"strconv"
+	"strings"
+)
+
+const minNumberOfLetters = 3
```

Comment:
Although I do like the visibility of the variable being pushed here, I'm also divided between visibility and avoiding the use of global state. I'd say go with the latter unless you know of a better guidance.